example-sdl.json

```diff
@@ -1,7 +1,8 @@
 {
   "source": "./package.json",
   "scripts": {
-    "run": "esy '@example-sdl' x TestSdl"
+    "run": "esy '@example-sdl' x TestSdl",
+    "run-bytecode": "dune exec example-sdl/TestSdl.bc"
```

Comment:
Can we add the non-SDL example (`@example`) to be run on Azure CI? IE, adding `esy '@example' run-bytecode` here:
https://github.com/revery-ui/reason-skia/blob/ce4185d23f4af25519a0fc1b214bd844f49bf9a7/.ci/esy-build-steps.yml#L25

Having it run on CI would help verify that it works cross-platform, and that we don't break the bytecode build in the future

Reply:
Sure thing! Do you want me to also set up CI for it, or should that be in a separate PR?

Reply:
Let's go ahead and add it in this PR - it should be as simple as adding:
```
  - script: esy @example run-bytecode
    displayName: 'esy @example run-bytecode
```
To the `.ci/esy-build-steps.yml`